libraries/freertos_plus/aws/ota/include/aws_iot_ota_agent.h

```diff
@@ -401,6 +401,9 @@ typedef struct
 typedef struct OTA_FileContext
 {
     uint8_t * pucFilePath; /*!< Local file pathname. */
+#if defined(__CC_ARM)
```

Comment:
Can we enable this somewhere else like main.c instead of OTA library source?

Reply:
@pvyawaha , to enable it in main.c can't be effective in the whole project and there are several ota common source code using aws_iot_ota_agent.h, such like as aws_iot_ota_agent.c, aws_iot_ota_mqtt.c.  More, this modification is for Keil compiler and not specific for some demo or some vendor. Refer to [Keil doc #pragma anon_unions](https://www.keil.com/support/man/docs/armcc/armcc_chr1359124984759.htm)

Reply:
@cyliangtw, we would want to keep the OTA library platform independent, and instead add platform specific logic in config files. Can you please consider adding the pragma logic within `aws_ota_agent_config.h` and `FreeRTOSConfig.h`  files within the `vendors/nuvoton` directory?

Reply:
@aggarw13 ,  thanks of your advisement and I moved to enable in nuvoton config files. So, I removed the previous commit of OTA lib and added one new commit of nuvoton config files.